src/lib.rs

```diff
@@ -274,10 +292,11 @@ where
     }
 }
 
-impl<S, W> Layer<S> for HierarchicalLayer<W>
+impl<S, W, FT> Layer<S> for HierarchicalLayer<W, FT>
 where
     S: Subscriber + for<'span> LookupSpan<'span>,
     W: for<'writer> MakeWriter<'writer> + 'static,
+    FT: FormatTime + 'static, // TODO(TmLev): Maybe it doesn't have to be static?
```

Comment:
Also unsure about `'static` lifetime here.

Reply:
We can always remove it in the future. So let's leave it in

Reply:
Removed `TODO`.